stdlib/mime/src/main/java/org/ballerinalang/mime/util/MimeUtil.java

```diff
@@ -433,14 +434,22 @@ static boolean isNestedPartsAvailable(BMap<String, BValue> bodyPart) {
     }
 
     /**
-     * Create ballerina error struct.
+     * Create mime specific error record.
      *
      * @param context Represent ballerina context
-     * @param errMsg  Error message in string form
-     * @return Ballerina error struct
+     * @param reason  Error code in string form
+     * @param errMsg  Actual error message
+     * @return Ballerina error record
      */
-    public static BError createError(Context context, String errMsg) {
-        return BLangVMErrors.createError(context, errMsg);
+    public static BError createError(Context context, String reason, String errMsg) {
```

Comment:
Since reason is a single value within MIME package, we can remove "reason" parameter and hard-code it